recipe/meta.yaml

```
@@ -63,6 +61,7 @@ outputs:
         - ambertools >=20
         - {{ pin_subpackage('openff-toolkit-base', exact=True) }}
         - notebook
+	- ipywidgets >=7,<8
```

Tabs vs. spaces maybe? I've never seen whitespace in an added line show up as red before
```suggestion
        - ipywidgets =7
```
This should be the same thing, a little simpler

---

Nice. And I gotta update my emacs config to deal with those tabs!